From 5a2db801c6520be296cee9cba0e0e4ffac68430c Mon Sep 17 00:00:00 2001
From: Bartosz Golaszewski <brgl@bgdev.pl>
Date: Wed, 11 May 2022 15:11:19 +0200
Subject: [PATCH] setup: don't use setuptools-markdown

This project is deprecated and irrelevant for the functionality of
pyrlp. We don't support it in meta-python so just drop it from the
dependencies.

Signed-off-by: Bartosz Golaszewski <brgl@bgdev.pl>
---
 setup.py | 2 --
 1 file changed, 2 deletions(-)

diff --git a/setup.py b/setup.py
index 1055fb1..55fca24 100755
--- a/setup.py
+++ b/setup.py
@@ -46,13 +46,11 @@ setup(
     # *IMPORTANT*: Don't manually change the version here. See README for more.
     version='3.0.0',
     description="A package for Recursive Length Prefix encoding and decoding",
-    long_description_markdown_filename='README.md',
     author="jnnk",
     author_email='jnnknnj@gmail.com',
     url='https://github.com/ethereum/pyrlp',
     packages=find_packages(exclude=["tests", "tests.*"]),
     include_package_data=True,
-    setup_requires=['setuptools-markdown'],
     install_requires=[
         "eth-utils>=2.0.0,<3",
     ],
-- 
2.34.1

